engine/testabilities/testmode/db_mode.go

```diff
@@ -0,0 +1,45 @@
+/*
+Package testmode provides functions to set special modes for tests,
+allowing to use actual Postgres or SQLite file for testing, especially for development purposes.
+Important: It should be used only in LOCAL tests.
+Calls of SetPostgresMode and SetFileSQLiteMode should not be committed.
+*/
+package testmode
+
+import (
+	"os"
+	"testing"
+)
+
+const (
+	modeEnvVar = "TEST_DB_MODE"
+	nameEnvVar = "TEST_DB_NAME"
+)
+
+// SetPostgresMode sets the test mode to use actual Postgres and sets the database name.
+func SetPostgresMode(t testing.TB, dbName string) {
```

Comment:
Maybe the name should suggest that it shouldn't be committed in actual tests.
`DevelopmentOnlyMode_SetPostgresMode` or something :shrug:

Reply:
It is under `testmode` package. 
I think it's enough.